examples/jsm/renderers/common/CanvasRenderTarget.js

```diff
@@ -0,0 +1,187 @@
+import { DepthTexture, Vector4, REVISION, createCanvasElement } from 'three';
+
+class CanvasRenderTarget {
+
+	constructor( parameters ) {
+
+		this.isCanvasRenderTarget = true;
+
+		this.canvas = parameters.canvas;
+		this.context = parameters.context;
+		this.domElement = parameters.domElement;
+		this.alpha = ( parameters.alpha === undefined ) ? true : parameters.alpha;
+
+		this.antialias = ( parameters.antialias === true );
+
+		if ( this.antialias === true ) {
+
+			this.sampleCount = ( parameters.sampleCount === undefined ) ? 4 : parameters.sampleCount;
+
+		} else {
+
+			this.sampleCount = 1;
+
+		}
+
+		this.samples = this.sampleCount;
+
+		this.depthTexture = new DepthTexture();
+
+		this.depth = true;
+		this.stencil = true;
+
+		this._width = 0;
+		this._height = 0;
+		this.pixelRatio = 1;
+
+		this.viewport = new Vector4( 0, 0, this._width, this._height );
+		this.scissor = new Vector4( 0, 0, this._width, this._height );
+		this._scissorTest = false;
+
+	}
+
+	getDomElement() {
```

Comment:
`getDOMElement` maybe? Or maybe we can implement it just via a getter?

Reply:
I've added a getter to match Renderer()